backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Helpers/PolicyHelper.cs

```diff
@@ -108,7 +108,7 @@ public static string GetAltinnAppsPolicyPath(string resourceRegistryId)
                 throw new ArgumentException("ResourceRegistryId was not defined");
             }
 
-            return $"{resourceRegistryId.AsFileName()}/policy.xml";
+            return $"{resourceRegistryId.AsFileName()}/resourcepolicy.xml";
```

Comment:
The function name should be renamed from GetAltinnAppsPolicyPath to GetResourceRegistryPolicyPath.
Documentation of resourceRegistryId property also needs update